pkg/sensors/tracing/generickprobe.go

```diff
@@ -1561,6 +1561,62 @@ func handleMsgGenericKprobe(m *api.MsgGenericKprobe, gk *genericKprobe, r *bytes
 			}
 			arg.Label = a.label
 			unix.Args = append(unix.Args, arg)
+		case gt.GenericKernelCap:
+			var output uint64
+			var arg api.MsgGenericKprobeArgKernelCapType
```

Comment:
the push 64-bit arg is a bit redundant it would be nice if we could extract that somehow to a helper. But I think the typing is a bit awkward in golang/here for that to work.